experimental/SetPartitions/src/SetPartition.jl

```diff
@@ -404,3 +404,109 @@ function compose_count_loops(p::SetPartition, q::SetPartition)
     
     return (ret, length(related_comp))
 end
+
+"""
+    number_of_blocks(V::SetPartition)
+
+Return the number of blocks of a `SetPartition`.
+
+# Examples
+```jldoctest
+julia> number_of_blocks(SetPartition([1, 2, 3], [2, 1, 3, 3]))
```

Comment:
```suggestion
julia> number_of_blocks(set_partition([1, 2, 3], [2, 1, 3, 3]))
```
use the lowercase constructor in doctests